maven_plugin/src/main/java/io/github/chains_project/maven_lockfile/FreezeDependencyMojo.java

```diff
@@ -41,41 +42,75 @@ public class FreezeDependencyMojo extends AbstractMojo {
     @Parameter(defaultValue = "${session}", readonly = true, required = true)
     private MavenSession session;
 
+    @Parameter(defaultValue = "pom.lockfile.xml", property = "pomLockfileName")
+    private String pomLockfileName;
```

Comment:
```suggestion
    @Parameter(defaultValue = "pom.lockfile.xml", property = "pomLockfileOutput")
    private String pomLockfileOutput;
```

Just following convention as most CLIs have the output flag to create an output file.